lib/middleware/karma.js

```diff
@@ -179,6 +179,10 @@ var createKarmaMiddleware = function (
             var filePath = file.path
             var fileExt = path.extname(filePath)
 
+            if (!files.included.hasOwnProperty(i)) {
+              continue
```

Comment:
I guess that depends on style. The style here seemed to be to have all the variable declarations at the top of the block, so I added my code underneath. 

None of the existing variable initializations should have a problem dealing with an undefined value, but it could break future code. I could go either way.